### PR TITLE
Add quadlet container support for Mask,Umask options

### PR DESCRIPTION
**Pull request by rhatdan**
#### Does this PR introduce a user-facing change?

```release-note
Quadlet containers now support the Mask and Unmask options.
```

### CODE REVIEW

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

**Comment by rhatdan** [comment]
@containers/podman-maintainers PTAL

File: docs/source/markdown/podman-systemd.unit.5.md [hunk 1]
```diff
@@ -584,6 +588,16 @@ entry from the unit file takes precedence
...
 
+### `Unmask=`
+
+Specify the paths to unmask separated by a colon. unmask=ALL or /path/1:/path/2, or shell expanded paths (/proc/*):
+
+If set to `ALL`, Podman will unmask all the paths that are masked or made read-only by default.
+
+The default masked paths are /proc/acpi, /proc/kcore, /proc/keys, /proc/latency_stats, /proc/sched_debug, /proc/scsi, /proc/timer_list, /proc/timer_stats, /sys/firmware, and /sys/fs/selinux.
+
+The default paths that are read-only are /proc/asound, /proc/bus, /proc/fs, /proc/irq, /proc/sys, /proc/sysrq-trigger, /sys/fs/cgroup.
```
Comment (vrothberg) [inline docs/source/markdown/podman-systemd.unit.5.md:599]: I am a bit worried about the redundancy between the quadlet man page and the podman ones.  It's not maintainable to have too many copies.  Can't we point to podman man pages for the details instead?

Let's also mention what the counterparts on the podman CLI are.

**Review by vrothberg — COMMENTED** [review]
Code LGTM but we should think about how to reduce redundancy/copies in the man pages.  Maybe @edsantiago has ideas?  The `@options` for Podman really improved the quality of the man pages.

File: docs/source/markdown/podman-systemd.unit.5.md (hunk 1 above)
Comment (rhatdan) [reply docs/source/markdown/podman-systemd.unit.5.md:599]: Yes I agree, I was first looking at using @openions , but they actually we too specific to podman commands.  I think we need to break these down one more layer.

**Pull request merged**